go.mod

```diff
@@ -5,12 +5,12 @@ go 1.18
 require (
 	github.com/VictoriaMetrics/fastcache v1.10.0
 	github.com/ava-labs/avalanche-network-runner v1.2.0
-	github.com/ava-labs/avalanchego v1.7.16
+	github.com/ava-labs/avalanchego v1.7.18
 	github.com/cespare/cp v0.1.0
 	github.com/creack/pty v1.1.18
 	github.com/davecgh/go-spew v1.1.1
 	github.com/deckarep/golang-set v1.8.0
-	github.com/ethereum/go-ethereum v1.10.20
+	github.com/ethereum/go-ethereum v1.10.21
```

Comment:
I don't think we have merged geth 1.10.21 or any recent coreth versions yet. Seems that we're still on 1.10.18:
https://github.com/ava-labs/subnet-evm/blob/master/params/version.go#L36

Reply:
They are merged. That just wasn't updated.